### PR TITLE
How to build a simple web service client. [AD-2101]

### PR DESCRIPTION
**Pull request by mkohl**

### CODE REVIEW

**Comment by BirgitBader** [comment]
@mkohl Did you find someone for a review from the technical perspective?

File: soap/tutorial-build-webservice-client.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# 1. How to integrate web services in your own project.
+
+Go to the epages-soapclient repository at https://bintray.com/epages/maven/epages-soapclient/ and integrate it with your build automation tool or manually.
+
+## 1.1 Integrate it with your build automation tool.
+
+Include bintray to your maven repository resolution.
+For gradle this might look like this:
+
+{% highlight bash %}
```
Comment (dpauli) [inline soap/tutorial-build-webservice-client.md]: Should it be:
```bash ?

File: soap/tutorial-build-webservice-client.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+## Product creation
+
+Create one or more products.
+First create an object of the type TCreate_Input with your new data.
+The values for TaxClass and alias are required.
+
+{% highlight java %}
+TCreate_Input createInput = new TCreate_Input();
+craeteInput.setAlias("fancyAlias");
+createInput.setTAxClass("/TaxMatrixGermany/normal");
```
Comment (dpauli) [inline soap/tutorial-build-webservice-client.md]: Perhaps the reader should know, that this tax class needs to be exist.

Comment (mkohl) [inline soap/tutorial-build-webservice-client.md]: done

File: soap/tutorial-build-webservice-client.md (hunk 1 above)
Comment (mkohl) [inline soap/tutorial-build-webservice-client.md]: No, see also https://github.com/mkohl/epages-docs/blob/develop/soap/how-to-soapclient.md. Also this documentory is not for Github. It is shown here: https://developer.epages.com/soap/ with different markup.

**Pull request merged**